app/jobs/update_ticket_links_job.rb

```diff
@@ -31,14 +37,27 @@ def relink_tickets(before_sha, after_sha)
       ticket.paths.each do |feature_review_path|
         next unless feature_review_path.include?(before_sha)
 
-        link_feature_review_to_ticket(ticket.key, feature_review_path, before_sha, after_sha)
+        update_path_and_link_feature_review_to_ticket(ticket.key, feature_review_path, before_sha, after_sha)
       end
     end
   end
 
-  def link_feature_review_to_ticket(ticket_key, old_feature_review_path, before_sha, after_sha)
+  def check_branch_nane_for_ticket_and_link(full_repo_name, branch_name, after_sha)
+    ticket_key = extract_ticket_key_from_branch_name(branch_name)
+
+    post_not_found_status(full_repo_name, after_sha) && return unless ticket_key
```

Comment:
Could also be
```suggestion
    return post_not_found_status(full_repo_name, after_sha) unless ticket_key
```

Reply:
Is this usual for ruby? I'm a n00b so this might not be a valid view, but `return post_not_found_status` makes me thing that `post_not_found_status` returns something that our method is also interested in returning, which it's not.

Reply:
Depends what behaviour you want. As it stands if `post_not_found_status` returns `nil` then there will not be an early return.

Reply:
In Ruby a method will return the return value of the last evaluated statement. I don't think it particularly matters here what the method is returning. If you want it to return `nil` here then it would need to be
```suggestion
    unless ticket_key
      post_not_found_status(full_repo_name, after_sha)
      return
    end 
```

Reply:
Fixed 👍